### PR TITLE
Configure mergify to squash commit instead of merge. 

### PR DESCRIPTION
**Pull request by ldjebran**
# Description
In the context to simplify git logs, and to escape some problems faced when merging a PR with many commits
- configure mergify to squash commit instead of merging
- configure mergify commit message to be the first commit message

do it in 2 commits to test the behavior. 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by omaciel — APPROVED** [review]
/lgtm

**Comment by ldjebran** [comment]
@Mergifyio refresh

**Comment by ldjebran** [comment]
@Mergifyio queue edge-api

File: .github/mergify.yml [hunk 1]
```diff
@@ -96,7 +96,8 @@ pull_request_rules:
           - invalid
       queue:
         name: edge-api
-        method: merge
+        method: squash
+        commit_message_template: "{{ commits[0] }}"
```
Comment (de1987) [inline .github/mergify.yml:100]:
```suggestion
        commit_message_template: {{ commits[0] }}
```
I think should be without quotes

**Review by de1987 — CHANGES_REQUESTED** [review]
should it be like I suggested?

File: .github/mergify.yml (hunk 1 above)
Comment (ldjebran) [reply .github/mergify.yml:100]: it does not work wihout quotes

**Review by de1987 — APPROVED** [review]

**Review by acosferreira — APPROVED** [review]
lgtm

**Comment by loadtheaccumulator** [comment]
@mergify watcha waitin' for?

**Review by lahavyuv86 — APPROVED** [review]
lgtm

**Pull request merged**